test/datahike/test/upsert_implem_test.cljc

```diff
@@ -37,17 +37,17 @@
         (let [new [5 :name "Jo" 3]
               tree-after (upsert-helper tree new)]
           ;; Is new in?
-          (is (= new (msg/lookup tree-after new)))))
+          (is (= [new nil] (first (msg/lookup-fwd-iter tree-after new))))))
```

Comment:
Why do you have to use the iterator now?

Reply:
Because `lookup`returns the value and not the key, and with our latest change we are not storing the value anymore, so I can no longer use `lookup`.